.vsts-ci/linux.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
   - powershell: |
       Get-ChildItem -Path env:
-    displayName: Capture environment
+    displayName: Capture Environment
```

Comment:
@RDIL Before we continue please move all style changes to another PR. Thanks.

Reply:
done

Reply:
I still see many style changes. Please revert all.

Reply:
That is all the style issues, rest are functionality changes